lib/server.js

```diff
@@ -313,7 +313,7 @@ function Server (options) {
     return c
   }
 
-  function newConnection (conn) {
+  self.newConnection = function (conn) {
```

Comment:
Why does this need to be `self.newConnection`?

Reply:
It just needs to be a public function. We should probably move it to the prototype.

Reply:
I can throw together a simple example for the documentation.